spec/ddtrace/contrib/rspec/instrumentation_spec.rb

```diff
@@ -5,36 +5,35 @@
 require 'ddtrace'
 
 RSpec.describe 'RSpec hooks' do
-  extend ConfigurationHelpers
-
   let(:configuration_options) { {} }
 
-  around do |example|
-    old_configuration = ::RSpec.configuration
-    ::RSpec.configuration = ::RSpec::Core::Configuration.new
-
+  before do
     Datadog.configure do |c|
       c.use :rspec, configuration_options
     end
-    example.run
+  end
+
+  # Yields to a block in a new RSpec global context. All RSpec
+  # test configuration and execution should be wrapped in this method.
```

Comment:
can this be done on an `around` block to avoid having to do it manually for each test?

Reply:
To be honest, I like the explicitness -- we're running RSpec to test RSpec so having clear boundaries on when transitions between the nested under-test RSpec and the top-level running-test RSpec happen I think is good for avoiding confusion.

Reply:
Adding it to `around` limits the use of the host RSpec test suite features inside the test example.